github-pullrequest-plugin/src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRCheckName.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.github.pullrequest;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import hudson.Extension;
+import hudson.model.Describable;
+import hudson.model.Descriptor;
+import jenkins.model.Jenkins;
+
+/**
+ * Represents the name of a check on a PR. In the GitHub API, this is referred to as the "context"
+ * of a commit status. This is the identifier used by GitHub to identify this type of check across
+ * different pull requests. For example, this could be "unit-tests" to indicate that the commit
+ * status refers to the outcome of running the unit tests against a particular commit.
+ *
+ * @author Sean Gilhooly
+ */
+public class GitHubPRCheckName implements Describable<GitHubPRCheckName> {
```

Comment:
More right it called commit status context that not really tied to PR because status context is also used for branches.